### PR TITLE
Feature/week2

### PR DESCRIPTION
**Pull request by mariblan**
Add some styling with a external css file

### CODE REVIEW

File: index.html
Comment (adichats92) [inline index.html:1]: Add an unordered list with the names of all the participants of the class.

**Review by adichats92 — CHANGES_REQUESTED** [review]

**Review by CiaranW12 — CHANGES_REQUESTED** [review]
header > nav > ul > li > a:hover {
  font-size: 2.0rem;
}

Please change font size on hover to 1.7rem ---  2.0's too big

**Review by MAMCB — CHANGES_REQUESTED** [review]
Add a hover effect to the anchor elements in the nav in main i.e:
 main>nav a:hover{add some effect}

File: Primer/html and css lectures/styles.css [hunk 1]
```diff
@@ -13,28 +55,26 @@ nav {
...
-nav ul li {
+main > nav ul li {
   display: inline-block;
   margin: 1px 10px;
   padding: 2px;
 }
-/* nav a {
-  margin: 10px;
-  padding: 10px;
-  background-color: rgb(144, 227, 238);
```
Comment (AnnaHovzan) [inline Primer/html and css lectures/styles.css]: could you make:  background-color: rgb(82, 204, 221);

File: Primer/html and css lectures/styles.css [hunk 2]
```diff
@@ -13,28 +55,26 @@ nav {
...
-  border: 1px solid black;
-} */
+
+/* NAVBAR */
+
+/* SECTIONS AND ARTICLES */
+
 h1 {
   font-size: 40px;
-  color: blueviolet;
```
Comment (AnnaHovzan) [inline Primer/html and css lectures/styles.css:37]: and could you change: color: rgb(76, 24, 124);

**Review by AnnaHovzan — CHANGES_REQUESTED** [review]
could you change a color?

File: index.html
Comment (ValentijnVermeer) [inline index.html:1]: Please add link to contactus.html next to the aboutus button.

File: Primer/html and css lectures/styles.css
Comment (ValentijnVermeer) [inline Primer/html and css lectures/styles.css:1]: Please add a small margin to all sections.

**Review by holc7 — CHANGES_REQUESTED** [review]
Could you align h1, h2 in the center by adding "text-align: center;" ?

**Comment by SabinaMB** [comment]
Change h2<Who are we and what are we doing?> in h2<Who are we and what are we creating?>

**Comment by naeemsajjad1** [comment]
<h2>image tags</h2>

**Review by naeemsajjad1 — CHANGES_REQUESTED** [review]
the image word in h2 is wrongly written
      <h2>image tags</h2>

**Comment by ValentijnVermeer** [comment]
Please add a small margin to all sections.

**Review by ValentijnVermeer — CHANGES_REQUESTED** [review]
Please add a small margin of 2em to all sections.

**Review by ValentijnVermeer — CHANGES_REQUESTED** [review]
Please add link to contactus.html next to the aboutus button.

**Review by MAMCB — APPROVED** [review]

**Review by adichats92 — APPROVED** [review]

**Review by AnnaHovzan — APPROVED** [review]

**Review by CiaranW12 — APPROVED** [review]

**Review by MihaelaWBS — APPROVED** [review]

**Review by ValentijnVermeer — APPROVED** [review]

**Review by naeemsajjad1 — APPROVED** [review]

**Pull request merged**